apps/sms/views/inbox/js/startup.js

```diff
@@ -47,6 +47,10 @@
         initLazyDependencies();
       });
 
+      InboxView.once('fully-loaded', () => {
+        App.setReady();
```

Comment:
It's not related to this particular issue, right, but it's required for the integration test that we have for split view mode - https://github.com/azasypkin/gaia/commit/7aa2b76b0fbbbda38b4809c69ecb652080f30e26#diff-17bd958c31567384687de6f1e8680881R82

> BTW do you think we'll need unit test for it?

Not sure, I'll check if we have one for non-split mode and do the same here as well :)